waterdata/templates/monitoring_location.html

```diff
@@ -46,12 +48,7 @@ <h1>{{ stations[0].station_nm }}</h1>
                     <li><a href="{{ config.NWIS_ENDPOINTS.UV }}/?cb_00060=on&format=gif_mult_parms&site_no={{ stations[0].site_no }}&period=7" target="_blank">Interactive grapher</a></li>
                     <li><a href="{{ config.NWIS_ENDPOINTS.UV }}/?format=img_default&site_no={{ stations[0].site_no }}&period=7" target="_blank">Presentation Charts</a></li>
                 </ul>
-                <div class="wdfn-component" data-component="hydrograph" data-siteno="{{ stations[0].site_no }}">
-                    <div class="compare-container" data-siteno="{{ stations[0].site_no}}">
-                        <input ga-on="click" ga-event-category="TimeseriesGraph" ga-event-action="toggleCompare" id="show-last-year-input" class="hydrograph-last-year-input" type="checkbox" value="show-last_year" />
-                        <label for="show-last-year-input">Show last year</label>
-                    </div>
-                </div>
+                {{ components.TimeseriesComponent(stations[0].site_no) }}
```

Comment:
I'd be okay with putting the floodSlider and map in a macro as you have done for the timeseries graph

Reply:
OK